spec/models/manageiq/providers/vmware/infra_manager/vm_spec.rb

```diff
@@ -43,14 +38,35 @@
   end
 
   context "supports_clone?" do
-    let(:ems)  { FactoryGirl.create(:ems_vmware) }
-    let(:vm)   { FactoryGirl.create(:vm_vmware, :ext_management_system => ems) }
-
     it "returns true" do
       expect(vm.supports?(:clone)).to eq(true)
     end
   end
 
+  context("supports_shutdown_guest?") do
+    let(:vm_status) { {:raw_power_state => power_state_suspended, :tools_status => 'toolsNotInstalled'} }
```

Comment:
I have no strong opinion on this, but I'd rather try to have the `expect` always in the `it` block. This makes it way easier to grasp the whole example at once. In this case I have to wrap my head around `assert_operation_availability` and `let(:vm_status)`. Imho this doesnt justify the 2 lines of code you are saving.